.github/workflows/website.yml

```diff
@@ -0,0 +1,18 @@
+name: publish_mkdocs_site
+on:
+  pull_request:
+    branches: [master]
+    paths: ['docs/**']
```

Comment:
I do not think that we need to deploy the site when opening a pull request cause it may contain bad information. I suggest leaving only push trigger

Reply:
Sure this is only for testing